libevm/VMConfig.h

```diff
@@ -22,17 +22,28 @@ namespace eth
 
 ///////////////////////////////////////////////////////////////////////////////
 //
-// interpreter configuration macros for optimizations and tracing
+// interpreter configuration macros for development, optimizations and tracing
+//
+// EIP_615                - subroutines and static jumps
+// EIP_616                - SIMD
 //
 // EVM_SWITCH_DISPATCH    - dispatch via loop and switch
 // EVM_JUMP_DISPATCH      - dispatch via a jump table - available only on GCC
 //
-// EVM_USE_CONSTANT_POOL  - 256 constants unpacked and ready to assign to stack
+// EVM_USE_CONSTANT_POOL  - constants unpacked and ready to assign to stack
 //
-// EVM_REPLACE_CONST_JUMP - with pre-verified jumps to save runtime lookup
+// EVM_REPLACE_CONST_JUMP - pre-verified jumps to save runtime lookup
 //
 // EVM_TRACE              - provides various levels of tracing
 
+#ifndef EIP_615
+	#define EIP_615 false
```

Comment:
Presumably these will stay off in production until EIPs are accepted, and then be removed.  So in the meantime it's good to be able to switch configurations with minimal recompilation.  That's my main consideration.  cmake files might work. Rebuilding with with cmake at the tope would not.